ratis-server/src/main/java/org/apache/ratis/server/impl/ServerProtoUtils.java

```diff
@@ -384,22 +385,29 @@ static InstallSnapshotRequestProto toInstallSnapshotRequestProto(
             .addAllFileChunks(chunks)
             .setTotalSize(totalSize)
             .setDone(done);
+    // Set term to DEFAULT_TERM as this term is not going to used by installSnapshot to update the RaftConfiguration
+    final LogEntryProto confLogEntryProto = toLogEntryProto(raftConfiguration, DEFAULT_TERM,
+        raftConfiguration.getLogEntryIndex());
```

Comment:
The term of this LogEntryProto is not used to update the in memory RaftConfiguration (or ConfigurationManager). But it is updated in the Raft Meta conf file (raft-meta.conf).
@szetszwo , @lokeshj1703, would it be better to send the correct term corresponding to the last conf entry while sending the installSnapshot?